variables.tf

```diff
@@ -4,7 +4,8 @@
 
 variable "resource_group_id" {
   type        = string
-  description = "The resource group ID where resources will be provisioned."
+  description = "The resource group ID where resources will be provisioned. It is required for creating the Cloud Object Storage instance."
```

Comment:
The resource group ID where The COS instance will be provisioned. It is required for creating the Cloud Object Storage instance. It is required if setting input variable `create_cos_instance` to `true`